### PR TITLE
see all articles on home page

### PR DESCRIPTION
**Pull request by Klnder**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <Header />
+      <main>
+        <Routes>
+          <Route index path="/home" element={<Home />} />
+          <Route path="/home/:article_id" element={<Article />} />
```
Comment (RichardNeat) [inline src/App.jsx:18]: you might want to have this url be more reflective of the url we navigate from, for example if /myarticles is where all articles are rendering, then maybe /myarticles/:article_id would be more appropriate for a single article as we've just come from there, if that makes sense :)

Also technically we don't need this at all anyway currently as it's not relevant to the current ticket.

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import HomeArticle from "../components/HomeArticle";
+import "./Home.css";
+
+function Home() {
+  const [articles, setArticles] = useState([]);
+
+  async function getArticles() {
```
Comment (RichardNeat) [inline src/pages/Home.jsx:9]: Ideally we should have our api calls in their own file to keep our components tidy and allow our api calls to be reusable if we need them again.

**Review by RichardNeat — COMMENTED** [review]
Nice work, I've left some suggested changes in comments below. I would also add:

- Currently when I navigate to your page on "/" I get your 404 page rendering, we should have something rendering there rather than that being an invalid route.
- Can we change our divs to something more semantic throughout the app? It's always good to follow good semantic html conventions throughout for accessibility purposes.

File: src/App.jsx (hunk 1 above)
Comment (Klnder) [reply src/App.jsx:18]: myarticles will be the url to see the articles the user posted, but it seems ambigous so will think of another name :)

**Pull request merged**